aca-project/src/main/com/aca/homework/week7/list/empty/FixedSizeList.java

```
@@ -0,0 +1,35 @@
+package com.aca.homework.week7.list.empty;
+
+public class FixedSizeList implements List {
+
+    private Object[] objects = new Object[10];
+    private int elem;
+
+    @Override
+    public int size() {
+        return elem;
+    }
+
+    @Override
+    public void add(Object o) {
+        if (elem == objects.length) {
+            System.out.println("The list is full");
+            return;
+        }
+        objects[elem] = o;
+        elem++;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return elem == 0;
+    }
+
+    @Override
+    public Object get(int index) {
+        if (index < 0 || index >= this.elem) {
+            return -1;
```

let's throw an exception instead of returning -1. maybe someone added Integer -1 objects to our list. why are we returning an invalid object